docs/source/modin/indexing.rst

```diff
@@ -62,7 +62,6 @@ Index
     Index.is_boolean
     Index.is_floating
     Index.is_integer
-    Index.is_interval
```

Comment:
why remove it?

Reply:
I removed it because I accidentally added this a long time ago -- this is not in the scope of the APIs we wanted to implement, even before we reduced the number of Index APIs to be implemented

Reply:
We don't generally include docstring stubs for methods we haven't implemented, but I think we should keep it in the supported doc.

Reply:
We don't generally include docstring stubs for methods we haven't implemented, but I think we should keep it in the supported doc.

Reply:
make sense. Let's at least keep it in the supported doc then.

Reply:
I added it back